### ISSUE TITLE
Split out elementary logos into their own repository/package

### ISSUE THREAD

**Issue by cassidyjames** [elementary/icons#377]
>If one doesn't take care to fix this downstream, it's entirely possible to unintentionally use the elementary logo in places they shouldn't be used.
>
>To avoid this, the elementary logos should be removed from this repository and put into their own overlay package that elementary OS uses. That way, when the theme is packaged, the logo doesn't get included unless someone deliberately adds it.

–@Conan-Kudo from https://github.com/elementary/stylesheet/issues/247

**Comment by danrabbit** [comment]
Yeah, it would make it easier for our downstreams and potentially for OEMs that want to provide a remix OS if we pulled `distributor-logo` out here. I think the easiest thing to do (without having to create another repo/package/etc) would be just to put it in our debian packaging
